docs/Documentation/Development/Build_Tools/git.md

```diff
@@ -101,12 +101,12 @@ Users already have SSH keys created on the HPC systems. To set up Github SSH aut
     ```
 
 ??? note "Revert to a previous commit"
```

Comment:
The title of this note should probably be updated as well.  Could say either "Undo a previous commit" or "Revert a previous commit" (as opposed to reverting *to* a previous commit).